tests/vmi_lifecycle_test.go

```diff
@@ -432,9 +406,9 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				})
 
 				It("[test_id:1630]should log warning and proceed once the secret is there", func() {
-					userData := fmt.Sprintf("#!/bin/sh\n\necho 'hi'\n")
 					userData64 := ""
-					vmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), userData)
+					vmi = libvmi.NewCirros()
+					vmi.Namespace = testsuite.GetTestNamespace(vmi)
```

Comment:
It does not work without this one?
it should because `RunVMIAndExpectScheduling` is suppose to add a default one if not already defined.

Reply:
Yes you're right - it was a leftover - thanks @EdDev